packages/sdk-react/src/components/auth/Auth.tsx

```diff
@@ -84,10 +61,10 @@ const Auth: React.FC<AuthProps> = ({
   const [passkeyCreated, setPasskeyCreated] = useState(false);
 
   const handleResendCode = async () => {
-    if (step === "otpEmail") {
-      await handleOtpLogin("EMAIL", email, "OTP_TYPE_EMAIL");
-    } else if (step === "otpPhone") {
-      await handleOtpLogin("PHONE_NUMBER", phone, "OTP_TYPE_SMS");
+    if (step === OtpStep.Email) {
+      await handleOtpLogin(FilterType.Email, email, "OTP_TYPE_EMAIL");
```

Comment:
enum-ify `OTP_TYPE_EMAIL` as well?

Reply:
This gets passed directly into the activity but we can just make this the enum